subprojects/core-api/src/main/java/org/gradle/api/internal/attributes/DescribableAttributesSchema.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.api.internal.attributes;
+
+import org.gradle.api.attributes.AttributesSchema;
+
+import java.util.List;
+
+public interface DescribableAttributesSchema extends AttributesSchema {
```

Comment:
Is the plan to move the methods in this interface up to the public interface eventually?

Reply:
Eventually, yes, probably not in this form.